sqlest/src/main/scala/sqlest/executor/Database.scala

```diff
@@ -171,7 +176,7 @@ class Session(database: Database) extends Logging {
   }
 }
 
-case class Transaction(database: Database) extends Session(database) {
+case class Transaction(database: Database) extends Session(database) with sqlest.extractor.ExtractorSyntax[ResultSet] {
```

Comment:
Now that `RowCountAndKeys` is being removed, `ExtractorSyntax` can also be removed